it/controllers/employment/EmploymentSummaryControllerISpec.scala

```diff
@@ -68,6 +68,19 @@ class EmploymentSummaryControllerISpec extends IntegrationTest with ViewHelpers
         welshToggleCheck(ENGLISH)
       }
 
+      "redirect when there is no data" which {
```

Comment:
We have a similar test (Line 189) where IncomeTaxUserData() is completely empty. This test seems to be based on the hmrc segments in the IncomeTaxUserData being empty. Maybe rename this test so the distinction is clearer.

Reply:
Updated to better reflect the scenario